project/src/test/kotlin/org/ionproject/core/calendar/CalendarControllerTest.kt

```diff
@@ -128,283 +66,279 @@ RRULE:FREQ=WEEKLY;UNTIL=20190610T000000Z;BYDAY=TH
 END:VEVENT
 END:VCALENDAR"""
 
-    val classCalendarCal4j = buildCalendarForClass()
-    val classSectionCalendarCal4j = buildCalendarForClassSection()
```

Comment:
What is the reason for these changes?

Reply:
The only change that occurred here was the identation, since lines 69 and 70 add the same code snippet